demos/smart_classroom_demo_gapi/main.cpp

```diff
@@ -547,6 +556,42 @@ bool ParseAndCheckCommandLine(int argc, char *argv[]) {
 }
 
 }  // namespace
+std::unique_ptr<AsyncDetection<detection::DetectedObject>> face_detector;
+namespace custom {
+    std::string getBinPath(const std::string & pathXML) {
+        std::string pathBIN(pathXML);
+        return pathBIN.replace(pathBIN.size() - 3, 3, "bin");
+    }
+    /**Nets**/
+    G_API_NET(Faces, <cv::GMat(cv::GMat)>, "face-detector");
+    G_API_OP(PostProc, <cv::GArray<detection::DetectedObject>(cv::GMat, cv::GMat)>, "custom.fd_postproc") {
+        static cv::GArrayDesc outMeta(const cv::GMatDesc &, const cv::GMatDesc &) {
+            return cv::empty_array_desc();
+        }
+    };
+    /**PostProcKernels**/
+    GAPI_OCV_KERNEL(OCVPostProc, PostProc) {
+        static void run(const cv::Mat &in_ssd_result,
+            const cv::Mat &in_frame,
+            std::vector<detection::DetectedObject> &out_faces) {
+
+            out_faces = face_detector->fetchResults(in_ssd_result, in_frame);
+        }
+    };
+
+    G_TYPED_KERNEL(ToRGB, <cv::GMat(cv::GMat)>, "custom.to_rgb") {
+        static cv::GMatDesc outMeta(const cv::GMatDesc &in) {
+            return in;
+        }
+    };
+    
+    GAPI_OCV_KERNEL(OCVToRGB, ToRGB) {
```

Comment:
If it valid conversion to RGB Mat for network so face-detection has the same output.